modules/canvaskit/README_BS.md

```diff
@@ -0,0 +1,90 @@
+# Bluescape Extra Notes
+
+# Installation Emscripten
+For emscripten installation use version 2.0.14
+Python 3.0 may not work as described on the website so make it via python 2.7
+So the workable sequence of commands is:
+
+	git clone https://github.com/emscripten-core/emsdk.gi
```

Comment:
Looks like you are missing a 't' at the end of the URL. It should be '.git' instead of '.gi'

Reply:
:)